frontend/src/components/core/Sidebar/Sidebar.tsx

```diff
@@ -189,7 +189,7 @@ class Sidebar extends PureComponent<SidebarProps, State> {
           </StyledSidebarCloseButton>
           <SidebarNav
             appPages={appPages}
-            hasSidebarElements={hasElements}
+            hasSidebarElements={!!hasElements}
```

Comment:
Is the `!!` related to the typescript checks? `hasElements` should be a boolean anyways, or?

Reply:
Ah, this was previously done because `hasElements` was an optional prop due to the types on `ThemedSidebar` and `Sidebar` being a bit too loose. I've since made them more precise, so this can be removed.